### ISSUE TITLE
Tizen 2017 misreports support for persistent licenses

### ISSUE THREAD

**Issue by joeyparrish** [shaka-project/shaka-player#894]
```
Safari 3.0.0 (Tizen 3.0.0) Offline stores, plays, and deletes protected content FAILED
	Failed: Shaka Error DRM.LICENSE_RESPONSE_REJECTED (Updating session failed. (99999))
	Error: Shaka Error DRM.LICENSE_RESPONSE_REJECTED (Updating session failed. (99999))
	    at new shaka.util.Error (lib/util/error.js:65:13)
	    at null.<anonymous> (lib/media/drm_engine.js:1106:24)
```

It would appear that Tizen TVs claim support for persistent licenses, but do not actually support them.  We should:

1. File a bug on Samsung to fix the platform to report accurately
2. Work around the bug by detecting this lack of support in `DrmEngine.probeSupport`

**Comment by joeyparrish** [comment]
Tizen 3.0 web app user agent string, for reference:

`Mozilla/5.0 (SMART-TV; LINUX; Tizen 3.0) AppleWebKit/538.1 (KHTML, like Gecko) Version/3.0 TV Safari/538.1`

**Comment by joeyparrish** [comment]
In the original failing test ("Offline stores, plays, and deletes protected content"), the platform doesn't fail the persistent license flow until the license response is given to the CDM.

We now have an additional failing test (new test added in #878):

```
Offline
    ✗ stores, plays, and deletes protected content with a temporary license
	Error: Timeout - Async callback was not invoked within timeout specified by jasmine.DEFAULT_TIMEOUT_INTERVAL.

Safari 3.0.0 (Tizen 3.0.0): Executed 1 of 1221 (1 FAILED) (skipped 1220) ERROR (2 mins 4.54 secs / 2 mins 0.032 secs)
```

With logging turned on, the new failure shows the error "Must not be destroyed", which is ambiguous.

**Comment by joeyparrish** [comment]
The new test is failing because `DrmEngine.prototype.createOrLoad` returns a Promise that is never resolved.  `this.allSessionsLoaded_` for temporary sessions is only resolved through key status events.  We don't have those on 2017 Tizen TVs.  (#891)

**Comment by joeyparrish** [comment]
Work-arounds are in place now, but we still have not been able to file bugs against Samsung/Tizen.  Having trouble getting an account on their JIRA instance.

**Comment by joeyparrish** [comment]
Work-arounds have been cherry-picked and will be released in v2.1.5.

**Comment by joeyparrish** [comment]
Samsung will never fix this, so there's no need for us to continue tracking it.  I'm closing it now.
